src/openfl/display/DisplayObject.hx

```diff
@@ -1653,7 +1653,8 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable #if openf
 		
 		if (value != __rotation) {
 			
-			__rotation = value;
+			__rotation = __normalizeAngleAS3(value);
```

Comment:
I think we should normalize before the comparison, so we don't do extra calculations if we somehow pass the same angle but unnormalized.